srcpkgs/brother-hl5340d-lpr/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'brother-hl5340d-lpr'
+pkgname=brother-hl5340d-lpr
+_br_version=2.0.3
+_br_revision=1
+version=${_br_version}
+revision=1
+archs="i686 x86_64"
+create_wrksrc=yes
+make_dirs="/var/spool/lpd 0775 root lp"
+depends="a2ps ghostscript"
+short_desc="LPR driver for the brother HL5340D printer"
+maintainer="Monkey 7<monkeyseven@fblog.ch>"
+license="brother-eula GPL-2"
```

Comment:
Use SPDX identifiers, also I think Brother EULA makes this package `nonfree=yes`